addons/l10n_it_edi_sdicoop/models/account_edi_format.py

```diff
@@ -80,6 +88,9 @@ def _cron_receive_fattura_pa(self):
                 except AccountEdiProxyError as e:
                     _logger.error('Error while receiving file from SdiCoop: %s', e)
 
+            if retrigger:
+                self.env.ref('l10n_it_edi.ir_cron_receive_fattura_pa_invoice')._trigger()
```

Comment:
self.env.ref('l10n_it_edi.ir_cron_receive_fattura_pa_invoice') != self?

Reply:
Yes. tested: `self == account.edi.format()`, so empty recordset.
This is a cron, it's not called on an instance of a model.
And it's called on the EDI format model.
*(I adapted this bit of code from the code of the other EDI cron. )*